joomla/joomla-cms
# [5.4] Task Scheduler require custom cron execution rules to be filled out

Pull Request for Issue #37313

### Summary of Changes

When you create a new scheduled task and select custom execution rules, and you don't specify a execution time, you can't save it now

### Testing Instructions

1. Create Demo Task - Sleep
Select Cron Expression Advanced as Execution Rule
Do not select all values in area Cron match
Try to save, Joomla will show you that you still have empty fields
2. Change execution rule to something else than Cron Expression Advanced and check if that is still working as before.

### Actual result BEFORE applying this Pull Request

1. You were able to click save, but then ran into an error

### Expected result AFTER applying this Pull Request

1. You see that there are required fields left and you cannot save
2. Other execution rules than Cron Expression Advanced are still working.

### Documentation Changes Required

None
